src/core/middlewares/sizequota/builder.go

```diff
@@ -119,6 +119,11 @@ func (*manifestCreationBuilder) Build(req *http.Request) (interceptor.Intercepto
 	// Replace request with manifests info context
 	*req = *req.WithContext(util.NewManifestInfoContext(req.Context(), info))
 
+	// Sync manifest layers to blobs for foreign layers not pushed and they are not in blob table
+	if err := info.SyncBlobs(); err != nil {
```

Comment:
Could we just log err instead of blocking PUT manifest? As all of the required information has been inserted into DB at this time. 

Sync err will not break any further action?

Reply:
change to only log the error